classes/dsr/README.md

```diff
@@ -116,6 +116,20 @@ You need the following third-party software:
       sudo ldconfig
 ```
 
+- pybind11 (used in DSR python wrapper) from https://github.com/pybind/pybind11 through the following steps :
```

Comment:
I'm testing if sudo apt install python3-pybind3 is an alternative to this.

Reply:
I didn't work on my side, maybe because I'm using Anaconda.
But I think that I have read some issues about using `apt install python3-pybind3` with CMake, too.